pennylane/ops/qutrit/__init__.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the discrete-variable qutrit quantum operations.
+
+The operations are in one file:
+
+* ``matrix_ops.py``: Generalized operations that accept a matrix parameter,
+  either unitary or hermitian depending.
+"""
+
+from .matrix_ops import *
+from ..identity import Identity
+
+ops = {
+    "Identity",
```

Comment:
Will it not be an issue that `Identity` is implemented such that it's a qubit operation (e.g., its matrix representation)?

Reply:
It will be in the future. However, currently there aren't any functions that explicitly rely on the matrix and/or eigenvalues of the operation/observable such as `expval` or `var`. This will need to be changed in the future, but for now, I will add a TODO, and I think we can leave it as is.